Final Readme.md file

iharshka opened the pull request:
Hey @arpitghura 
This is the final commit from my side, give your input also if there is a need for more changes to the readme.md file.
Regards.
Harsh

arpitghura commented on README.md:
```diff
@@ -1,16 +1,21 @@
 
-The Web App can be accessible from [https://typing-test-project.vercel.app/](https://typing-test-project.vercel.app/)
+The Web App can be accessible from [https://typing-test-project.vercel.app/](https://typing-test-project.vercel.app/)</br></br>
+![Typing test](https://img.shields.io/badge/Hacktoberfest-2022-blue?style=round-square&logo=Hacktoberfest&logoColor=black&link=https://hacktoberfest.com/)
```
Create all the badges from [https://shields.io/](https://shields.io/) and make badge in `for-the-badge` style. 

Use these credentials:
Username: `arpitghura`
repo name: `typing-test`

arpitghura reviewed (changes requested):
Make changes as requested and also create badges from the [https://shields.io/](https://shields.io/) and don't copy and paste the badges from some other repo.

iharshka commented on README.md:
```diff
@@ -1,16 +1,21 @@
+The Web App can be accessible from [https://typing-test-project.vercel.app/](https://typing-test-project.vercel.app/)</br></br>
+![Typing test](https://img.shields.io/badge/Hacktoberfest-2022-blue?style=round-square&logo=Hacktoberfest&logoColor=black&link=https://hacktoberfest.com/)
+![Typing test](https://img.shields.io/badge/Contribute-red?style=round-square&logo=Github&logoColor=black&link=https://github.com/arpitghura/typing-test)
+[![Typing Test](https://img.shields.io/badge/Typing-Test-brightgreen.svg?style=flat-square)](https://makeapullrequest.com)
```
Which one?

iharshka commented:
> Make changes as requested and also create badges from the https://shields.io/ and don't copy and paste the badges from some other repo.

I have just taken a reference, thanks for sharing the link.

iharshka commented:
Hey @arpitghura,
I have added the requested changes in this commit.
Changes- Moved the badges to the top, added for-the-badge style, added credentials, updated links of the badges.
Please check and do let me know.
regards

arpitghura commented on README.md:
```diff
@@ -24,18 +30,20 @@ The Web App can be accessible from [https://typing-test-project.vercel.app/](htt
 
+Thanks for coming here!
 ### Give a star if you liked this!
+#### Regards,</br>
```
Don't write these. 
Delete all after regards.

arpitghura reviewed (changes requested):
Delete after regards as taught

iharshka commented:
I have made the requested changes. Please check it @arpitghura

arpitghura reviewed (approved):
Good work

iharshka commented:
Thanks arpit
